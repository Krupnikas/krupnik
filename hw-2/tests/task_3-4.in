4
12.02131 4599497 0.000112 123.0135
