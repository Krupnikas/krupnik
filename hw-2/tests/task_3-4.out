4599632.034922
